revolv/payments/migrations/0003_auto_20150103_0323.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+def populate_data(apps, schema_editor):
+    PaymentInstrumentType = apps.get_model("payments", "PaymentInstrumentType")
+
+    PaymentInstrumentType.objects.create(name="reinvestment")
+    PaymentInstrumentType.objects.create(name="check")
```

Comment:
I was thinking either a docstring or the readme, cause it's directly related to the code. And if readme, I think we should have a reference somewhere in the docstring for PaynentTransactionType anyway